src/views/outbound/RequestBuilder.jsx

```diff
@@ -1460,7 +1460,7 @@ class RequestBuilder extends React.Component {
         // } else {
         //   delete this.props.request.params
         // }
-        this.headerBodyBuilderRef.current.updateBodyChanges();
+        // this.headerBodyBuilderRef.current.updateBodyChanges();
```

Comment:
Can you add some comment here stating the reason for disabling this line with story number?

Reply:
@vijayg10 done